### DISCUSSION TITLE
Generate plot of the results

### DISCUSSION THREAD

**Discussion by MagicHealer** [ContinualAI/avalanche#1295] [Q&A]
Hi, May I ask if how can I plot the results of the training and testing results using matplotlib? Thank you

**Comment by AndreaCossu** [comment]
The call to `strategy.train` or `strategy.eval` returns a dictionary of metrics that you can easily use for plotting. 
Check out the from [zero to hero tutorial on evaluation](https://avalanche.continualai.org/from-zero-to-hero-tutorial/05_evaluation#accessing-metric-values) for more info.
